code/game/gamemodes/changeling/powers/transform.dm

```diff
@@ -7,17 +7,138 @@
 	req_human = 1
 	max_genetic_damage = 3
 
+//changeling clothing items
+/obj/item/clothing/head/changeling

```

Comment:
I've decreed that this is garbage, please have only ONE flesh clothing type, and change the Slot ID on it as needed.